src/org/usfirst/frc/team467/robot/Ramp.java

```diff
@@ -18,14 +18,16 @@
 
 	private String name;
 	private State state;
+	private boolean exists;
 
 	public enum State {
 		UP,
 		DOWN;
 	}
 
-	public Ramp(String name, int forwardChannel, int reverseChannel) {
-		if (!RobotMap.HAS_RAMPS) {
+	public Ramp(String name, int forwardChannel, int reverseChannel, boolean exists) {
+		this.exists = exists;
+		if (!exists) {
```

Comment:
Log message should also say which ramp this is.

Reply:
I wonder if it would make sense to create a third state called NOT_EXIST?

Then I think none of the other code has to change.